### ISSUE TITLE
Store Finished Runs Count in Splits File

### ISSUE THREAD

**Issue by wooferzfg** [LiveSplit/LiveSplit#941]
I think it makes sense for it to be editable.

**Comment by wooferzfg** [comment]
@CryZe I think it would make sense to have this here and in livesplit-core. It seems similar to Subsplits since it is something that developed as a useful thing to have but we only support a hacky way of editing it.

**Comment by CryZe** [comment]
Yeah I agree. However that yet again clutters up the Run Editor even more, so how do you think we should put this in the Run Editor?

**Comment by wooferzfg** [comment]
Maybe just put it on the same line as Offset and Attempt Count? If we want to conserve space, we could just put Game Name and Category Name on the same line and then maybe make the whole window a bit wider.

It would be better to somehow combine Attempt Count and Finished Runs Count into one UI element, but I'm not sure how to do that without making it confusing.

**Comment by just-ero** [comment]
What's the status on this?

**Comment by wooferzfg** [comment]
Same status as before - not done yet
